docs/roadmap/roadmap.md

```diff
@@ -0,0 +1,13 @@
+# acs-engine Roadmap
+
+The acs-engine Roadmap is a community document created as part of the
+[Planning Process](planning-process.md). Each roadmap item describes a high-level capability or
+grouping of features that are deemed important to the future of acs-engine.
+
+Given the project's rapid [Release Schedule](releases.md), roadmap
+items are designed to provide a sense of direction over many releases.
+
+## CI dashboard available to the public
+## Native Azure VNET CNI support
+## Multi-GPU support
+## Support for dedicated etcd cluster VMs
```

Comment:
Per our discussion over using the wiki to track Roadmap, I'm going to delete this `roadmap.md` file.